docs/api/target/feature_correlation.rst

```diff
@@ -49,36 +52,39 @@ See `scikit-learn documentation <http://scikit-learn.org/stable/modules/generate
 
     visualizer = FeatureCorrelation(method='mutual_info-regression',
                                     labels=feature_names)
-    visualizer.fit(X, y, discrete_features=discrete_features, random_state=0)
+    visualizer.fit(X, y, discrete_features=discrete_features, random_state=42)
     visualizer.poof()
 
-.. image:: images/feature_correlation_mutual_info_regression.png
 
 Mutual Information - Classification
 -----------------------------------
 
 By fitting with a pandas DataFrame, the feature labels are automatically obtained from the column names.
 This visualizer also allows sorting of the bar plot according to the calculated mutual information (or Pearson correlation coefficients) and selecting features to plot by specifying the names of the features or the feature index.
 
-.. code:: python
+.. plot::
+    :context: close-figs
+    :alt: FeatureCorrelation on the diabetes dataset using mutual_info-classification
 
     from sklearn import datasets
     from yellowbrick.target import FeatureCorrelation
+    import pandas as pd
+    import numpy as np
 
     # Load the regression data set
-    data = datasets.load_diabetes()
+    data = datasets.load_wine()
     X, y = data['data'], data['target']
     feature_names = np.array(data['feature_names'])
     X_pd = pd.DataFrame(X, columns=feature_names)
 
     feature_to_plot = ['alcohol', 'ash', 'hue', 'proline', 'total_phenols']
 
+
     visualizer = FeatureCorrelation(method='mutual_info-classification',
                                     feature_names=feature_to_plot, sort=True)
-    visualizer.fit(X_pd, y, random_state=0)
+    visualizer.fit(X_pd, y, random_state=42)
```

Comment:
for my understanding @Kautumn06 do we want to change random_state values where they are already set?  random_state=42 does bring unity across the docs

Reply:
@Kautumn06 suggested that we should change to random_state=42 for consistency.

Reply:
Hi @pdamodaran, In #775 I recommended setting the `random_state=42` when the parameter had not previous been set to a specific value. However, in this case `random_state` had already been set to `0` so it should not be changed.

Reply:
@Kautumn06 - I have changed this back to random_state = 0